lib/cisco_node_utils/interface.rb

```diff
@@ -74,8 +74,6 @@ def access_vlan
 
     def access_vlan=(vlan)
       config_set('interface', 'access_vlan', @name, vlan)
-    rescue Cisco::CliError => e
-      raise "[#{@name}] '#{e.command}' : #{e.clierror}"
```

Comment:
In one provider I actually had created a method for the raise instead of using the pattern on every setter. 
But I agree it would be better to enhance CliError, assuming it somehow has name as part of the context.